README.md

```diff
@@ -189,7 +194,7 @@ Encoding of the `key` and `value` objects will adhere to `'keyEncoding'` and `'v
 --------------------------------------------------------
 <a name="get"></a>
 ### db.get(key[, options][, callback])
-<code>get()</code> is the primary method for fetching data from the store. The `key` can be an arbitrary data object. If it doesn't exist in the store then the callback will receive an error as its first argument. A not-found err object will be of type `'NotFoundError'` so you can `err.type == 'NotFoundError'` or you can perform a truthy test on the property `err.notFound`.
+<code>get()</code> is the primary method for fetching data from the store. The `key` can be an arbitrary data object. If it doesn't exist in the store then the callback or promise will receive an error as its first argument. A not-found err object will be of type `'NotFoundError'` so you can `err.type == 'NotFoundError'` or you can perform a truthy test on the property `err.notFound`.
```

Comment:
Reads as "[the] promise will receive an error as its first argument". Perhaps simply say: "If it doesn't exist then the callback or promise will receive an error."

Reply:
addressed